actix-http/src/body/utils.rs

```diff
@@ -7,71 +7,184 @@ use futures_core::ready;
 
 use super::{BodySize, MessageBody};
 
-/// Collects the body produced by a `MessageBody` implementation into `Bytes`.
+/// Collects all the bytes produced by `body`.
 ///
 /// Any errors produced by the body stream are returned immediately.
 ///
+/// Consider using [`to_bytes_limit`] instead to protect against memory exhaustion.
```

Comment:
I wonder if marking it as `deprecated` would be a stronger/better way to deliver the message, especially to those who are _today_ using `to_bytes` and exposed to memory exhaustion.

Reply:
Hmm, maybe in a later release we could deprecate it?